packages/context/src/__tests__/unit/inject.unit.ts

```diff
@@ -271,3 +272,76 @@ describe('property injection', () => {
     expect(meta.foo.metadata).to.eql({x: 1, decorator: '@inject'});
   });
 });
+
+describe('inspectTargetType', () => {
+  it('handles static method injection', () => {
```

Comment:
This belongs to https://github.com/strongloop/loopback-next/pull/5455 I think

Reply:
Yes. just rebased.